### ISSUE TITLE
Try to catch the "Rate limit exceeded" error and display something more useful

### ISSUE THREAD

**Issue by Daniel-Mietchen** [WDscholia/scholia#302]
Sometimes, loading a Scholia aspect yields a 
```
Rate limit exceeded
```
message for one or more of the panels.

The problem is typically resolved by just reloading the page a few moments later but there may be something we can do to reduce the likelihood of that to happen, e.g. by adding a small random delay between the SPARQL calls for a Scholia page.

**Comment by fnielsen** [comment]
Another way would be to delay the embedding, only display the component when the user clicks

**Comment by Daniel-Mietchen** [comment]
The reason for this is that too many requests are being made within a certain time frame.

Several options to reproduce &mdash; key thing is to run multiple different queries at overlapping times.

For instance, you could

1. choose a panel with lots of links to richly annotated topics, e.g. https://scholia.toolforge.org/venue/Q2000008#topics-header
1. quickly right-click on multiple Scholia links (e.g. the entries in the Topics column in the example table) to open new tabs with Scholia profiles for those topics.
1. check these topic profile pages for Server errors, some or all of which may come with a "Rate limit exceeded" message.

Another situation in which this often occurs is by right-clicking the Random button at https://scholia.toolforge.org/author/ multiple times in quick succession.

**Comment by jdcaballerov** [comment]
@Daniel-Mietchen I think this PR https://github.com/WDscholia/scholia/pull/1652 will help a lot with this issue

**Comment by Daniel-Mietchen** [comment]
I like the simplicity but have my doubts how far this will move things forward, since (i) lazy load needs to be enabled, (ii) it does not address the issue of multiple WDQS iframes loading in parallel, (iii) it does not catch the error.

I just tried this under Firefox with lazy load enabled and still got "Rate limit exceeded" errors in the scenarios listed above. I did not see any such errors for loading just a single page though, and this is indeed progress.
